### ISSUE TITLE
[Enhancement] Merge calls and texts into one thread instead of creating new rooms

### ISSUE THREAD

**Issue by illgitthat** [mautrix/gvoice#3]
Currently, every time you receive a call a new matrix room is created.

Potentially relevant code: https://github.com/mautrix/gvoice/blob/main/pkg/libgv/gvproto/threads.pb.go#L32

`ThreadFolder_ALL_CALL_THREADS`

**Comment by Fmstrat** [comment]
Agreed, my server is flooded with rooms now. Would be better if all calls from an individual showed up in the same room as SMS.

**Comment by clari7744** [comment]
agreed with fmstrat; i'd like it to show up as "call" in the SMS thread - if it's possible to include elapsed time as well that'd be cool otherwise just having it show up as a call and ending it there rather than starting a new room

**Comment by illgitthat** [comment]
@KishanBagaria tulir mentioned you are planning a fix for this. Sharing my branch just in case it helps. I won't plan to PR mine as I bet you'll likely have a more elegant solution. https://github.com/illgitthat/gvoice/pull/1/changes/8d1f97be35f93d8dd85034d8e439f5880db0360f

thank you!
